src/lib/validation/form/no-csv-external-datasets.js

```
@@ -0,0 +1,27 @@
+const CSV_SRC_PREFIX = 'jr://file-csv';
+
+module.exports = {
+  requiresInstance: false,
+  skipFurtherValidation: false,
+  execute: async({ xformPath, xmlDoc }) => {
+    const errors = [];
+
+    const csvSources = getNodes(xmlDoc, `${XPATH_MODEL}/instance[@src]`)
+      .map(instance => instance.getAttribute('src').trim())
+      .filter(src => src.startsWith(CSV_SRC_PREFIX));
+    if(csvSources.length) {
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)